package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@movesfinancial/unit-node-sdk",
-  "version": "0.8.17-fork.8",
+  "version": "0.8.17-feat-add-received-payment-resource.4",
```

Comment:
can you revert this change?

Reply:
and then bump when you release on `fork`

Reply:
:+1: